gprofiler/utils/__init__.py

```diff
@@ -267,15 +282,28 @@ def run_process(
 
 
 def pgrep_exe(match: str) -> List[Process]:
-    pattern = re.compile(match)
-    procs = []
-    for process in psutil.process_iter():
-        try:
-            if pattern.match(process_exe(process)):
-                procs.append(process)
-        except psutil.NoSuchProcess:  # process might have died meanwhile
-            continue
-    return procs
+    if is_windows():
+        pythoncom.CoInitialize()
+        w = wmi.WMI()
+        return [
+            Process(pid=p.ProcessId)
+            for p in w.Win32_Process()
+            if match in p.Name.lower() and p.ProcessId != os.getpid()
+        ]
+    else:
+        pattern = re.compile(match)
+        procs = []
+        for process in psutil.process_iter():
+            # Added to prevent AttributeError on processes that have pid == 0
```

Comment:
Can this edit be removed now that we have a separate branch for windows?

Reply:
Done